gce_ovf_import_tests/test_suites/ovf_import_test_suite.go

```diff
@@ -322,30 +321,34 @@ func runOvfImportTest(
 		return
 	}
 
-	logger.Printf("[%v] Setting instance metadata with test startup script", testSetup.name)
-	err = client.SetInstanceMetadata(testProjectConfig.TestProjectID, testProjectConfig.TestZone,
-		instanceName, &api.Metadata{Items: []*api.MetadataItems{testSetup.startup},
-			Fingerprint: instance.Metadata.Fingerprint})
+	if testSetup.startup != nil {
```

Comment:
Minor: to be align with below coding style (if .... return), you may consider "if  testSetup.startup==nil ... Printf ... return"

Reply:
Changed